Doc/library/configparser.rst

```diff
@@ -1034,7 +1034,13 @@ ConfigParser Objects
       instance will contain an empty dataset.  An application which requires
       initial values to be loaded from a file should load the required file or
       files using :meth:`read_file` before calling :meth:`read` for any
-      optional files::
```

Comment:
The "::" here got lost.

Reply:
@Agent-Hellboy this is likely the source of your indentation error -- ``::`` denotes the start of a code block, and you've inserted a new paragraph in-between. You should move the code block or the `::``.

A

Reply:
ohh silly me.